omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirVersionUtils.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.web.servlet;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class FhirVersionUtils {
+	
+	public enum FhirVersion {
+		R4,
+		R3,
+		ERROR
```

Comment:
I would make this `UNKNOWN`. Also I would order the values in the enum this way: `UNKNOWN`, `R3`, `R4`. That way we're adding values to the end and (hopefully) not messing up the ordinal order of the enum.